roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
I personally hate having defaults set with blank or 'unset'. If I'm supposed to pass in a value for a variable, I'd prefer the role error with "One or more undefined variables" so that I know I need to fix something.

Otherwise the role completes 'successfully' but the service isn't properly setup. Then you have to go and hunt for the reason why.

So I like the idea of either having a sane default, or having no default and letting it error if they don't pass it in.